examples/dns/record.py

```diff
@@ -0,0 +1,26 @@
+from cloudflare import Cloudflare
+
+
+# define your variable here
+
+cloudflare_token = "your-cloudflare-token"
+cloudflare_zone_id = "zone"
+alias_name = "www.mydns.com"
+cannonical_name = "cname.example.com"
+
+# build cloudflare driver here
+client = Cloudflare(api_token=cloudflare_token)
```

Comment:
we shouldn't need to define `api_token` config here. if the value is defined in the environment, we pick that up.
```suggestion
client = Cloudflare()
```

Reply:
oh got it